<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- 加载参数 -->
  
  <param name="yolov5_path"       value="$(find yolov5_ros)/yolov5"/>
  <param name="use_cpu"           value="true" />

  <!-- 启动yolov5进行目标检测 -->
  <node pkg="yolov5_ros" type="yolo_v5.py" name="detection" output="screen" >
    <param name="weight_path"       value="$(find yolov5_ros)/weights/exp2/best.pt"/>
    <param name="image_topic"       value="robot_img" />
    <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="conf"              value="0.3" />
  </node>

  <!-- 启动中央控制系统 -->
  <node pkg="yolov5_ros" type="central_controller.py" name="central_controller" output="screen" />

  <!-- 启动机器人摄像头 -->
  <!-- <node pkg="yolov5_ros" type="robot_cam.py" name="robot_cam" output="screen" /> -->
  <node pkg="yolov5_ros" type="test.py" name="robot_cam" output="screen" />

   <!-- 启动机器人底层驱动包 -->
  <node pkg="yolov5_ros" type="robot_move_base.py" name="robot_move_base" output="screen" />

</launch>
